docs/core/diagnostics/runtime-events.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Runtime Events"
+description: Review events emitted by CoreCLR runtime.
+ms.date: "09/14/2020"
+ms.topic: reference
+helpviewer_keywords: 
+  - "runtime events [.NET Core]"
+  - "ETW, EventPipe runtime events (CoreCLR)"
+---
+# .NET Core runtime events
+
+For runtime events in .NET Framework, please refer to the [CLR ETW Events document](../../framework/performance/clr-etw-events.md).
```

Comment:
This intro paragraph should have more details as to why a user cares about this and what it's all about, not point them to another article.